pkg/convert/xmb.go

```diff
@@ -0,0 +1,71 @@
+package convert
+
+import (
+	"encoding/xml"
+	"fmt"
+
+	"go.expect.digital/translate/pkg/model"
+	"golang.org/x/text/language"
+)
+
+type xmb struct {
+	XMLName  xml.Name     `xml:"messagebundle"`
+	Language language.Tag `xml:"srcLang,attr"`
+	Messages []msg        `xml:"message"`
+}
+
+type msg struct {
+	ID          string `xml:"id,attr"`
+	Description string `xml:"desc,attr"`
+	Content     string `xml:"content"`
+	Fuzzy       bool   `xml:"fuzzy,attr"`
+}
```

Comment:
1) There is no `Language` in any form (element or attr) in XMB formatted XML, from my research
2) The resources are very sparse, but I found [example](https://localizify.com/formats/angular-xmb) of how XMB looks like, and it is kinda different from what you have now

Reply:
Here is another [example](https://github.com/angular/angular/issues/12441#issuecomment-261026476)

Reply:
After investigation: 
There is a lot of versions of XMB and which one is the correct we don't know for now. @janishorsts decided to put it on hold